README.md

```diff
@@ -41,7 +41,10 @@ const authenticator: GenesysCloudClientAuthenticator = authenticatorFactory(clie
   environment: 'mypurecloud.com',
   persist: false,
   storageKey: 'gc_client_auth_data',
-  debugMode: false
+  debugMode: false,
+  customHeaders: {
+    'X-My-Header': 'value'
+  }
```

Comment:
Two thoughts:
1. Since this isn't the default, it might be worth adding a comment here to make it clear we don't default to a header of `X-My-Header`, though I'll admit I would expect that to be pretty self-evident given the name. Or maybe just remove the comment above that says `these are the defaults`.
2. It could also be worth adding to `Popup Auth Usage` (but since it's an optional property, I also don't think it's a big deal for only one example to show it).

Reply:
I went ahead and remove the comment about that configuration being the defaults. I'm open to making any additional modifications to the code or docs, just let me know!